src/Lykke.Service.PayHistory/Filters/ErrorResponceExceptionFilterAttribute.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Common.Log;
+using Lykke.Common.Api.Contract.Responses;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace Lykke.Service.PayHistory.Filters
+{
+    public class ErrorResponceExceptionFilterAttribute : ExceptionFilterAttribute
```

Comment:
You should be using LykkeMiddleware instead because it already handles this case (LykkeMiddleware is included by default in Lykke Service template).